apps/frontend/utils/dropNotApproved.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  DroughtDto,
+  FloodDto,
+  IncidentDto,
+  ValidationStatusValue,
+} from '@wfp-dmp/interfaces';
+
+// drop the forms that have be flagged as not approved
+// Cannot be done in the backend because $ne doesn't work as a query option
```

Comment:
I think it is ok to do this in the frontend but this is not a good argument, we could filter after the query in the backend and return a filtered result